untidy/contaminators.py

```diff
@@ -1,9 +1,109 @@
 """ Functions to contaminate the data """
+import pandas as pd
+import numpy as np
+import random
 
 """ Column-wise contaminators """
 
+def data_check(data):
```

Comment:
Could we raname this to `get_data_dims()` or something specific along these lines